docs/docs/core/fallback-actions.mdx

```diff
@@ -6,117 +6,146 @@ description: Define custom fallback actions with thresholds for NLU and Core for
-the response `utter_ask_rephrase`, so be sure to specify this
+The default implementation of `action_default_ask_rephrase` utters
+the response `utter_ask_rephrase`, so make sure to specify this
 response in your domain file.
-The implementation of both actions can be overwritten with [custom actions](./actions.mdx#custom-actions).
+The implementation of both actions can be overwritten with
+[Custom Actions](./actions.mdx#custom-actions).
+
+To use the `Two-Stage-Fallback` for messages with low NLU confidence, add the
+following [Rules](./rules.mdx) to your training data. This rule will make sure that the
+`Two-Stage-Fallback` will be activated whenever a message is received with
+low classification confidence.
+
+```yaml
+rules:
+
+- rule: Implementation of the Two-Stage-Fallback
+  steps:
+  - intent: nlu_fallback
+  - action: two_stage_fallback
+  - form: two_stage_fallback
+```
+
+## Handling Low Core Confidence
+
+As users might send unexpected messages,
+it is possible that their behavior will lead them down unknown conversation paths.
+Rasa's machine learning policies such as the [TED Policy](./policies.mdx#ted-policy)
+are optimized to handle these unknown paths.
 
-You can specify the core fallback action as well as the ultimate NLU
-fallback action as parameters to `TwoStageFallbackPolicy` in your
-policy configuration file.
+To handle cases where even the machine learning policies can't predict the
+next action with high confidence, make sure to add the
+[RulePolicy](./policies.mdx#rule-policy) to your
+policy configuration. The [RulePolicy](./policies.mdx#rule-policy) will predict a
+default action if no [Policy](./policies.mdx) has a next action prediction with
+confidence above a configurable threshold.
+
+You can configure the action that is run in case low of Core confidence as well as
+the corresponding confidence threshold as follows:
 
 ```yaml
 policies:
-  - name: TwoStageFallbackPolicy
-    nlu_threshold: 0.3
-    core_threshold: 0.3
-    ambiguity_threshold: 0.1
-    fallback_core_action_name: "action_default_fallback"
-    fallback_nlu_action_name: "action_default_fallback"
-    deny_suggestion_intent_name: "out_of_scope"
+- name: RulePolicy
+  # Confidence threshold for the `fallback_action_name` to apply.
+  # The action will apply if no other action was predicted with
+  # a confidence >= core_fallback_threshold
+  core_fallback_threshold: 0.4
+  fallback_action_name: "action_default_fallback"
 ```
+
+`action_default_fallback` is a default action in Rasa Open Source that sends the
+`utter_default` response to the user. Make sure to specify
+the `utter_default` in your domain file. It will also revert back to the
+state of the conversation before the user message that caused the
+fallback, so it will not influence the prediction of future actions.
+You can take a look at the source of the action below:
+
+TODO!!!!
+.. autoclass:: rasa.core.actions.action.ActionDefaultFallback
```

Comment:
[currently not implemented](https://github.com/RasaHQ/rasa/issues/6194), so you might have to find a fallback here (no pun intended!)